test/parallel/test-cluster-fork-windowsHide.js

```diff
@@ -0,0 +1,39 @@
+'use strict';
+const common = require('../common');
+const assert = require('assert');
+const cluster = require('cluster');
+
+if (cluster.isMaster) {
+  const msg = 'foobar';
+
+  cluster.setupMaster({
+    silent: true,
+    windowsHide: true
+  });
+
+  const worker = cluster.fork();
+  const cstdin = worker.process.stdin;
+  const cstdout = worker.process.stdout;
+  let response = '';
+
+  worker.on('exit', common.mustCall((code, signal) => {
+    assert.strictEqual(code, 0);
+    assert.strictEqual(signal, null);
+  }));
+
+  cstdout.on('data', (data) => {
+    response += data.toString('utf8');
+
+    if (response === msg) {
+      worker.disconnect();
+    }
+  });
+  cstdin.write(msg, 'utf8');
```

Comment:
You don't have to specify 'utf8' if you're writing a string, that's the default encoding.

Reply:
I wasn't aware of that...